client/components/web-preview/content.jsx

```diff
@@ -114,6 +122,20 @@ export default class WebPreviewContent extends Component {
 		}
 	};
 
+	handleResize = () => {
+		const { vpw } = this.props;
+		const iframeScaleRatio = this.iframe.parentElement.clientWidth / vpw;
+		const iframeStyle = {
+			maxWidth: 'none',
```

Comment:
the none dynamic properties such as `max-width`, `margin` could ideally be moved to external scss